runtime/trappist/src/constants.rs

```diff
@@ -26,6 +26,7 @@ pub mod currency {
 	pub const CENTS: Balance = constants::currency::CENTS;
 	pub const GRAND: Balance = constants::currency::GRAND;
 	pub const MILLICENTS: Balance = constants::currency::MILLICENTS;
+	pub const DOLLARS: Balance = 100 * CENTS;
```

Comment:
To be removed, I don't think this is needed (see comments re. the pallet's config trait's parameters).

Reply:
Done in [33b1736](https://github.com/paritytech/trappist/pull/305/commits/33b173617174bd419a5246670ac3699375c25d51).